mlflow/store/db_migrations/versions/7f2a7d5fae7d_add_datasets_inputs_input_tags_tables.py

```diff
@@ -0,0 +1,79 @@
+"""add datasets inputs input_tags tables
+
+Revision ID: 7f2a7d5fae7d
+Revises: 3500859a5d39
+Create Date: 2023-03-23 09:48:27.775166
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from mlflow.store.tracking.dbmodels.models import SqlDataset, SqlInputTag, SqlInputs
+
+# revision identifiers, used by Alembic.
+revision = "7f2a7d5fae7d"
+down_revision = "3500859a5d39"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        SqlDataset.__tablename__,
+        sa.Column("dataset_uuid", sa.String(length=32), nullable=False),
+        sa.Column(
+            "experiment_id",
+            sa.Integer(),
+            sa.ForeignKey("experiments.experiment_id"),
+            primary_key=True,
+            nullable=False,
+        ),
+        sa.Column("name", sa.String(length=500), primary_key=True, nullable=False),
+        sa.Column("digest", sa.String(length=36), primary_key=True, nullable=False),
+        sa.Column("dataset_source_type", sa.String(length=36), nullable=False),
+        sa.Column("dataset_source", sa.String(length=5000), nullable=False),
+        sa.Column("dataset_schema", sa.String(length=5000), nullable=True),
+        sa.Column("dataset_profile", sa.String(length=5000), nullable=True),
```

Comment:
Just a reminder comment for our future selves: if we decide to use a larger string value for these columns (I think we should), then we should update the values in this migration script as well.

Reference: https://github.com/mlflow/mlflow/pull/8128/files#r1152772077